substrate/client/src/client.rs

```diff
@@ -136,6 +136,13 @@ impl<B, E> Client<B, E> where
 		})
 	}
 
+	/// Submit a transaction to this for potential inclusion in a block.
+	pub fn submit_transaction(&self, _tx: block::Transaction) -> error::Result<()> {
+		// TODO: introduce a tx queue and place the transaction in there. tx queue or some of its
```

Comment:
1. I don't think this belongs in client
2. substrate blocks don't necessarily have transactions, so this is something that belongs in chain-specific code (tx ordering is completely chain dependent)

Reply:
Can be removed now.

Reply:
@gavofyork this is dead code now, we should remove it